widget/embedded/src/components/NoResult/NoResult.styles.ts

```diff
@@ -10,6 +10,10 @@ export const Container = styled('div', {
 
 export const Footer = styled('div', {
   width: '100%',
+  '& ._typography': {
```

Comment:
I thought having this is better in expandable mode. If it is not possible please revert it.

Reply:
i revert it